specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzeconversations-authoring.json

```diff
@@ -1932,27 +1939,27 @@
     },
     "ConversationalAnalysisAuthoringExportedProjectFormat": {
       "enum": [
-        "clu",
-        "luis"
+        "Conversation",
+        "Luis"
```

Comment:
I think Hazem suggested all uppercase LUIS in his email?

Reply:
Hazem's comments were case-insensitive. He said that he'll leave the casing to us. And according to the Azure guideline, acronyms should be treated as normal words while defining their casing. E.g., Luis and Url. https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#172-casing

Reply:
I see. Then I need to update it in runtime.

Reply:
Technically, I believe those only apply to models and property names - not values. @tg-msft?

Reply:
It does say that all acronyms should be treated as normal words, then the next point says that everything should be lowerCamelCase. But since Text Analytics and CLU runtime both use PascalCase, the following is following the same casing here.

Reply:
Yes, property names in JSON should be `camelCased` but values can be whatever's natural for customers and the domain.  Both should be processed with case sensitivity by the service.  `LUIS` is fine if you think your customers will know what that means.  If you don't want everyone (including me) always wondering what the `I` stands for, you could use a value like `LanguageUnderstanding` instead.